src/index/getFilePaths.ts

```diff
@@ -18,30 +17,33 @@ const globSearch = (pattern: string) => {
   return files;
 };
 
-/** Recursively get all file paths. */
-const getFilePaths = (rootPath: string) => {
+export const getFilePaths = (rootPath: string) => {
   const filePaths: string[] = [];
   const paths = [rootPath];
 
   while (paths.length > 0) {
     const filePath = paths.shift();
-
-    if (filePath == null || filePath.length === 0) continue;
+    if (filePath === undefined || filePath.length === 0) break;
```

Comment:
Yes. `filePath == null` checks for both `undefined` and `null`.

Reply:
@sQVe Do we keep this how it is or accept the change?

Reply:
Well I don't really see why checking for both `null` and `undefined` here. If it's to check falsy values we must use `filePath || ...` instead.

Reply:
@AnatoleLucet Did you mean `!filePath || ...`? I agree.

Reply:
Yep, sorry 🙃

Reply:
Just checking for a falsy value will make this not work for paths like `"0"`, which is a falsy value...

Checking for both `null` and `undedined` is sane as it pops an array. That array could have `null` values in. Checking for `null` then guards from a application crash.

Reply:
The only possible falsy value in a path (as you said) is `0`. But [readdir & readdirSync](https://nodejs.org/api/fs.html#fs_fs_readdir_path_options_callback) returns an array of strings, and `!!"0"` is `true`. It would have to pass through a parse int to be falsy.
So I think it's better to check for falsy values with `!filePath || ...` so we cover more than `null` and `undefined`. I don't see any reason not to.

Reply:
I prefer to be explicit when possible as it makes code easier to understand and more readable.